autoload/javacomplete/util.vim

```diff
@@ -3,7 +3,7 @@
 "
 " Utility functions
 
-" TODO: search pair used in string, like 
+" TODO: search pair used in string, like
```

Comment:
> Here and below there is some character at the end of string. Remove, please.

hi @artur-shaik I can not fix this issue,because I can not see the character;